Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License
 
-FROM envoyproxy/envoy-alpine:v1.8.0
+FROM envoyproxy/envoy-alpine:latest
```

Comment:
If we need to move past the `v1.8.0` tag, let's still use a specific SHA -- `latest` makes it hard to know exactly what we built a given Ambassador version against.